_includes/topnav.html

```diff
@@ -2,84 +2,82 @@
-	                	<a href="{{item.url | remove: "/"}}" class="NavLink">{{item.title}}</a>
-	                </li>
-	                {% else %}
-	                <li class="NavLinkLi">
-	                	<a href="{{item.url | remove: "/"}}" class="NavLink">{{item.title}}</a>
-	               	</li>
-	                {% endif %}
-	                {% endfor %}
-	                {% endfor %}
-	                <!-- entries with drop-downs appear here -->
-	                <!-- conditional logic to control which topnav appears for the audience defined in the configuration file.-->
-	                {% for entry in site.data.topnav.topnav_dropdowns %}
-	                {% for folder in entry.folders %}
-	                <li class="dropdown">
-	                    <a href="#" class="dropdown-toggle" data-toggle="dropdown">{{ folder.title }}<b class="caret"></b></a>
-	                    <ul class="dropdown-menu">
-	                        {% for folderitem in folder.folderitems %}
-	                        {% if folderitem.external_url %}
-	                        <li><a href="{{folderitem.external_url}}" target="_blank">{{folderitem.title}}</a></li>
-	                        {% elsif page.url contains folderitem.url %}
-	                        <li class="dropdownActive"><a href="{{folderitem.url |  remove: "/"}}">{{folderitem.title}}</a></li>
-	                        {% else %}
-	                        <li><a href="{{folderitem.url | remove: "/"}}">{{folderitem.title}}</a></li>
-	                        {% endif %}
-	                        {% endfor %}
-	                    </ul>
-	                </li>
-	                {% endfor %}
-	                {% endfor %}
-	                {% if site.feedback_disable == null or site.feedback_disable == false %}
-				{% include feedback.html %}
-			{% endif %}
+	<div class="top-nav-item">
+		<img class="mobile-nav-btn" src="{{ "images/icons/mobile-menu.svg" }}" alt="">
+	</div>
 
-	                <li>
-	                    <!--start search-->
-	                    <div id="search-demo-container">
-	                        <input type="text" id="search-input" placeholder="Search...">
-	                        <ul id="results-container"></ul>
-	                    </div>
-	                    <script src="{{ "js/jekyll-search.js"}}" type="text/javascript"></script>
-	                    <script type="text/javascript">
-	                            SimpleJekyllSearch.init({
-	                                searchInput: document.getElementById('search-input'),
-	                                resultsContainer: document.getElementById('results-container'),
-	                                dataSource: '{{ "search.json" }}',
-	                                searchResultTemplate: '<li><a href="{url}" title="{{page.title | replace: "'", "\"}}">{title}</a></li>',
-	                    noResultsText: 'No results',
-	                            limit: 10,
-	                            fuzzy: true,
```

Comment:
This is weird. It shows up correctly indented in Sublime.

<img width="736" alt="screen shot 2017-10-25 at 10 34 11 pm" src="https://user-images.githubusercontent.com/8000644/32036431-a493391c-b9d4-11e7-990e-f24f9212318d.png">